tasks/jsinspect.js

```diff
@@ -92,13 +97,17 @@ module.exports = function(grunt) {
       writableStream.on('finish', function() {
         done(taskSucceeded);
       });
+      writableStream.on('error', function() {
+        throw new Error('Can not write to ' + options.outputPath);
```

Comment:
As for me, it is not a big hint. Possible cases: wrong access rights, folder with the same name exists, intermediate folders were not created, some internal stream errors.
At best, total error handling should be provided, but this is a part of the separate request.